src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -127,17 +138,17 @@ export const SubmissionHeader: React.FC = () => {
                 <Flex
                   flexDirection="row"
                   justifyContent={[
-                    submission?.externalId ? "end" : "space-between",
+                    submission?.externalId && isLoggedIn
```

Comment:
This change is needed because we will display the logo on mobile for logged out users.